nipype/interfaces/spm/model.py

```diff
@@ -313,12 +313,10 @@ def _parse_inputs(self):
         return einputs
 
     def _list_outputs(self):
-        import scipy.io as sio
-
         outputs = self._outputs().get()
         pth = os.path.dirname(self.inputs.spm_mat_file)
         outtype = "nii" if "12" in self.version.split(".")[0] else "img"
-        spm = sio.loadmat(self.inputs.spm_mat_file, struct_as_record=False)
+        spm = load_spm_mat(self.inputs.spm_mat_file, struct_as_record=False)
```

Comment:
It would be cleaner to break out the functionality you have into its own function and put the try/catch logic here.

```suggestion
        try:
            spm = sio.loadmat(self.inputs.spm_mat_file, struct_as_record=False)
        except NotImplementedError:  # Matlab 7.3+
            spm = load_spm_mat(self.inputs.spm_mat_file)
```